data_subscriptions/worker/latest_ckan_activity.py

```diff
@@ -28,6 +28,7 @@ def __call__(self):
             self.current_offset += self.server_limit
             if self.has_reached_the_end(response):
                 break
+            RemoteCKAN.close(api)
```

Comment:
@sagargg can we implement using 

```
with RemoteCKAN(self.url, user_agent=...) as api:
    ...
```

Reply:
@anuveyatsu  I tried but tests are failing. 

<img width="1163" alt="Screen Shot 2020-08-14 at 12 23 25" src="https://user-images.githubusercontent.com/11631101/90220960-f513e000-de28-11ea-8ae7-157899aecc09.png">

Reply:
@sagargg ok, let's remove user agent then and merge this.

Reply:
@anuveyatsu  removed.